var/spack/repos/builtin/packages/py-rst2pdf/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyRst2pdf(PythonPackage):
+    """Convert reStructured Text to PDF via ReportLab.
+
+    The usual way of creating PDF from reStructuredText is by going through
+    LaTeX. This tool provides an alternative by producing PDF directly using
+    the ReportLab library."""
+
+    homepage = "https://rst2pdf.org/"
+    pypi     = "rst2pdf/rst2pdf-0.99.tar.gz"
+
+    version('0.99', sha256='8fa23fa93bddd1f52d058ceaeab6582c145546d80f2f8a95974f3703bd6c8152')
+
+    depends_on('python@3.6:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
```

Comment:
Missing build dep on `py-setuptools-scm`

Reply:
Ah, is that coming from this in `setup.py`:
```
    use_scm_version=True,
```
I wondered about that.

Reply:
I see it in `pyproject.toml`, which I did not look at until just now.